### PR TITLE
Add empty states for Vulnerability

### PR DESCRIPTION
**Pull request by jiridostal**
* Adds action handler for Rejected actions
* Adds "errors" stanza to data structure
* Adds some of proposed Empty states and stores them in a separate file
image
image
image
image

### CODE REVIEW

**Comment by karelhala** [comment]
@jiridostal can you provide screenshot please?

File: src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTable.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import routerParams from '../../../../Utilities/RouterParams';
-import { SortDirection, TableVariant } from '../../../../PresentationalComponents/Table';
+import { SortByDirection, Table, TableBody, TableHeader } from '@patternfly/react-table';
 import findIndex from 'lodash/findIndex';
 import propTypes from 'prop-types';
-import { RowLoader } from '../../../../Utilities/helpers';
-import { Table, TableHeader, TableBody, sortable, SortByDirection } from '@patternfly/react-table';
+import React, { Component, Fragment } from 'react';
+import { Pagination } from '../../../../PresentationalComponents/Pagination';
+import { TableVariant } from '../../../../PresentationalComponents/Table';
```
Comment (karelhala) [inline src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTable.js]: TableVariant should be part of `@patternfly/react-table`

**Comment by jiridostal** [comment]
Updated initial comment and removed the import, it was unused.

**Comment by karelhala** [comment]
Looks really nice! Thanks.

**Pull request merged**